lib/src/js/polymer_base_dart.html

```diff
@@ -19,6 +19,30 @@
       if (this.__dartDetached__) this.__dartDetached__();
     };
 
+    var originalDeserialize = Polymer.Base.deserialize;
```

Comment:
Don't they need to be there like the detached and the like? Otherwise the original polymer behavior wouldn't be preserved.